e2e/rbd.go

```diff
@@ -191,6 +191,15 @@ func validateRBDImageCount(f *framework.Framework, count int, pool string) {
 	}
 }
 
+func isKeyExist(err error, stdErr string) bool {
```

Comment:
I do not get what this function does... the name does not describe the contents?